stdlib/public/TensorFlow/TensorGroup.swift

```diff
@@ -40,21 +40,10 @@ public protocol TensorArrayProtocol {
 /// When a `TensorGroup` is returned as a result of a tensor operation, it is
 /// initialized with its tensor fields set to the tensor operation's tensor
 /// results.
-//
-// TODO: Implement `TensorGroup` derived conformances so that users don't have
-// to implement conformances themselves.
 public protocol TensorGroup : TensorArrayProtocol {
   /// The types of the tensor stored properties in this type.
   static var _typeList: [TensorDataType] { get }
 
-  /// An array of `nil`s with the same number of elements as `_outputTypeList`.
-  /// The `nil` represents unknown shape.
-  // TODO: This is a protocol requirement so that conformances can provide
-  // custom const-evaluable implementations. When the const-evaluator is
-  // powerful enough to evaluate the default implementation, remove this
-  // requirement.
-  static var _unknownShapeList: [TensorShape?] { get }
```

Comment:
Could you explain why this requirement was removed?

Reply:
I removed it based on [this comment](https://github.com/apple/swift/pull/22854#issuecomment-466731011) by @rxwei.

Reply:
Ah, thanks!